tests/app.notification.test.js

```diff
@@ -0,0 +1,75 @@
+import client from "socket.io-client"
+import chaiHttp from "chai-http"
+import chai, { expect, request } from "chai"
+import { io, users } from "../src/utils/socket.utils"
+import app from "../src/app"
+import { userLogin } from "./dammyData"
+
+chai.use(chaiHttp)
+
+describe("in app notification test", () => {
+  let clientSocket, serverClient;
+  before(async () => {
+    serverClient = request(app).keepOpen();
+    const userLogin = await serverClient.post("/api/v1/users/login").send({
+      email: "MANAGER@gmail.com",
+      password: "MANAGER2gmail"
+    })
+    const { port } = app.address();
+    console.log(`http://localhost:${port}`, port);
+    clientSocket = client.connect(`http://localhost:${port}`, {
```

Comment:
could you try to get these url from your dotenv because what happens when we are going to connect to anothe url